pkg/apiserver/devops/devops.go

```diff
@@ -57,37 +60,40 @@ func SearchPipelineRuns(req *restful.Request, resp *restful.Response) {
 		return
 	}
 
-	_ = resp.WriteAsJson(res)
+	resp.Header().Set(restful.HEADER_ContentType, "application/json")
+	resp.Write(res)
 }
 
-func GetPipelineRun(req *restful.Request, resp *restful.Response) {
+func GetPipeBranchRun(req *restful.Request, resp *restful.Response) {
 	projectName := req.PathParameter("projectName")
 	pipelineName := req.PathParameter("pipelineName")
 	branchName := req.PathParameter("branchName")
 	runId := req.PathParameter("runId")
 
-	res, err := devops.GetPipelineRun(projectName, pipelineName, branchName, runId, req.Request)
+	res, err := devops.GetPipeBranchRun(projectName, pipelineName, branchName, runId, req.Request)
 	if err != nil {
 		parseErr(err, resp)
 		return
 	}
 
-	_ = resp.WriteAsJson(res)
+	resp.Header().Set(restful.HEADER_ContentType, "application/json")
```

Comment:
not a good practice to directly hard coding `application/json` here, better to use a variable

Reply:
response ok.